log-collector-script/linux/eks-log-collector.sh

```diff
@@ -346,6 +346,20 @@ get_iptables_info() {
     ip6tables-save > "${COLLECT_DIR}"/networking/ip6tables-save.txt
   fi
 
+  if ! command -v ipvsadm && command -v ipset > /dev/null 2>&1; then
+    echo "IPVS Linux kernel module not installed" | tee ipvsadm.txt ipset.txt
+  else
+    # check that ip_vs module is loaded in get_modinfo()
+    try "collect ipvs information"
+    ipvsadm --save | tee -a "${COLLECT_DIR}"/networking/ipvsadm.txt && sed -i '1s/^/add \tproto \tvirt-server \tscheduler algthm \treal-server \n/' "${COLLECT_DIR}"/networking/ipvsadm.txt
```

Comment:
Could you explain more about why it need to sed but not keeping raw data?

Reply:
What if there already have old `"${COLLECT_DIR}"/networking/ipvsadm.txt` existed. Is it expected to have `tee -a` or maybe it should be `tee` instead?

Reply:
@guessi, I see... not making assumptions that there isnt an ipvsadm.txt existing prior in the collection_dir
- updating tee to overwrite rather than append